src/diffusers/image_processor.py

```diff
@@ -29,15 +29,31 @@
 PipelineImageInput = Union[
     PIL.Image.Image,
     np.ndarray,
-    torch.FloatTensor,
+    torch.Tensor,
     List[PIL.Image.Image],
     List[np.ndarray],
-    List[torch.FloatTensor],
+    List[torch.Tensor],
```

Comment:
Could I get a brief explanation of why this is changing? For NumPy and Torch inputs, we assume values to be in [0, 1] range no?

Reply:
torch.FloatTensor is deprecated 
https://discuss.pytorch.org/t/why-cannot-i-call-floattensor-on-a-float-tensor/137652/2